src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java

```diff
@@ -273,4 +284,142 @@ public static Object intersectProperties(Object sourceObject, Class<?> targetCla
     public static String getCurrentDate() {
         return new SimpleDateFormat("dd MMM yyyy").format(new Date());
     }
+
+    /**
+     * Formats message for names of parties that notifications will be sent to.
+     */
+    public static String getParties(CaseData caseData) {
```

Comment:
A lot of this code is good but just wondering if the Helper class is the best place to put it, maybe in a NotificationHelper class?

Reply:
Good shout, that's a better place for it